plugins/alerting/pkg/alerting/drivers/alerting_manager/options.go

```diff
@@ -16,4 +18,5 @@ type AlertingDriverOptions struct {
 	InternalRoutingKey string                               `option:"internalRoutingKey"`
 	AlertingOptions    *shared.AlertingClusterOptions       `option:"alertingOptions"`
 	Subscribers        []chan alertingClient.AlertingClient `option:"subscribers"`
+	TlsConfig          *tls.Config                          `option:"tlsConfig"`
```

Comment:
I **really** don't wanna be that guy but acronyms should be in consistent case -> TLSConfig